packages/indexer/src/transaction/watcher.ts

```diff
@@ -62,26 +72,62 @@ class TransactionWatcher {
 
   private async start() {
     if (this.pollerIntervalId !== undefined) {
-      const error = new Error(
+      logger.error(
         `Interval was ${
           this.pollerIntervalId
         } when starting ${this.account.toBase58()}`
-      )
-      logger.error(error.message);
-      throw error;
+      );
+      return Err({ type: WatcherBackfillError.StartedWithPollingIntervalSet });
     }
-    await this.backfillFromLatest();
+    await this.handleBackfillFromLatest();
     // TODO: add websocket for realtime updates (might be lossy, but would allow us to increase poll time meaning less rpc costs)
-    this.pollerIntervalId = setInterval(() => {
-      this.backfillFromLatest();
+    this.pollerIntervalId = setInterval(async () => {
+      await this.handleBackfillFromLatest();
     }, 10000);
+    return Ok(`successfully started watcher for ${this.account.toBase58()}`);
+  }
+
+  private async handleBackfillFromLatest() {
+    const backfillRes = await this.backfillFromLatest();
+    switch (backfillRes.success) {
+      case true:
+        {
+          if (this.account) {
+            logger.info(
+              `successfully ran backfill for acct: ${this.account.toBase58()}`
+            );
+          }
+          logger.info(`success message: ${backfillRes.ok}`);
+        }
+        break;
+      case false:
+        {
+          logger.error(
+            `error running backfill from latest: ${backfillRes.error.type}`
+          );
+        }
+        break;
+    }
   }
 
-  private async backfillFromLatest() {
-    if (this.stopped) return;
-    if (this.backfilling) return;
+  private async backfillFromLatest(): Promise<
```

Comment:
Ideally we want a concrete Result type here. In this case looks like it should be 
`Result<string, WatcherBackfillError>`